merge.go

```diff
@@ -26,11 +26,12 @@ func hasExportedField(dst reflect.Value) (exported bool) {
 }
 
 type Config struct {
-	Overwrite               bool
-	AppendSlice             bool
-	TypeCheck               bool
-	Transformers            Transformers
-	overwriteWithEmptyValue bool
+	Overwrite                    bool
+	AppendSlice                  bool
+	TypeCheck                    bool
+	Transformers                 Transformers
+	overwriteWithEmptyValue      bool
+	OverwriteSliceWithEmptyValue bool
```

Comment:
This needs to be un-exported (as we are using `WithOverrideEmptySlice`), same as `overwriteWithEmptyValue` is not exported